AllASPFuncs.py

```diff
@@ -659,16 +659,24 @@ def find_parq(ID, path = '/hpc/group/cosmology/OpenUniverse2024/roman_rubin_cats
     Find the parquet file that contains a given supernova ID.
     '''
     files = os.listdir(path)
+
+    star = True if len(str(ID)) == 11 else False
```

Comment:
This seems super-dangerous.  Why 11 characters?  What if there is whitespace?  This will be really difficult to debug.  At least document this check, and assign `11` to a variable name (this is an example of how variable naming is part of documentation.)